templates/status.html

```diff
@@ -47,13 +47,18 @@
 
         function main ()  {
             // list of supported websites
-            var sites = ["facebook", "twitter", "instagram", "github", "soundcloud", "pinterest", "tumblr", "behance", "gitlab"];
+            var sites = ["facebook", "twitter", "instagram", "github", "soundcloud", "pinterest", "tumblr", "behance", "gitlab", "openhub", "deviantart"];
 
             // create cards dynamically for each of the websites
             sites.forEach(website => {
-              $(".helper").append(`<div class="card">
-                <p><div class="tooltip"><i class="fa fa-${website}"></i><span class="tooltiptext">${website}</span></div>
-                <span id='${website}'><i class="fa fa-circle-o-notch fa-spin"></i></span></p>`)
+              if (website != "openhub")
```

Comment:
please dont change this code.  Instead, use CSS rules to load the OpenHub logo.  It will have a class `fa-openhub` that you can use.

Reply:
ok

Reply:
this is still here.